src/utils/contracts.js

```diff
@@ -160,31 +160,130 @@ export async function setupContracts(account, library, chainId, fetchId) {
     })
   }
 
-  const fetchAPY = async () => {
+  const _rebasingCreditsPerToken = async (block = 'latest') => {
+    if (block === 'latest' && ContractStore.rebasingCreditsPerToken) {
```

Comment:
N.B. this prevents the page from noticing a rebase without hard reload. This seems a fine behavior bc of their infrequency (couple times a day)